engine/src/index.d.ts

```diff
@@ -600,6 +600,7 @@ export class Layer {
   set_name(v: string): string;
   get_opacity(): number;
   set_opaciy(v: number): number;
+  set_opacity(v: number): number;
```

Comment:
I'm extremely confident that the misspelled API has never been used, since I would definitely have noticed the mistake, so I'm OK with removing it.